examples/minimum/main.tf

```diff
@@ -0,0 +1,29 @@
+resource "random_string" "master_username" {
+  length  = 6
+  special = false
+  upper   = false
+  numeric = false
+}
+
+resource "random_password" "master_password" {
+  length  = 16
+  special = false
+  upper   = false
+}
+
+module "complete_cluster" {
+  source                    = "./../../"
+  cluster_identifier        = local.cluster_name
+  availability_zones        = data.aws_availability_zones.available.names
+  identifier                = "${local.cluster_name}-instance"
+  instance_class            = "db.t3.medium"
```

Comment:
Can we have a default instance type so you don't have to specify it - by default the smallest instance type.